openassessment/templates/openassessmentblock/edit/oa_rubric_reuse.html

```diff
@@ -0,0 +1,41 @@
+{% load i18n %}
+{% spaceless %}
+<div id="openassessment_rubric_clone">
+    <h3>{% trans "Clone Rubric" %}<span id="rubric-reuse-collapse-icon"></span></h3>
+    <div>
+        <script type="text/template" id="openassessment_rubric_clone_other_ora_data">{{ other_openassessments_in_course | safe }}</script>
+        <div class="openassessment_rubric_clone_block" id="openassessment_rubric_clone_description">{% trans "To clone a rubric from an existing published or unpublished draft, you can search by the Block ID. Note that cloning is one-way, meaning changes made after cloning will only effect the rubric being modified, and will not modify not any rubric it was cloned from." %}</div>
+        <div class="openassessment_rubric_clone_block" id="openassessment_rubric_clone_block_id"><b>{% trans "Block ID For this ORA:" %}</b> {{ block_location }}</div>
```

Comment:
Proposed tweak:

If you add the following CSS to the "Block ID for this ORA" you can make it non-selectable. That lets an instructor triple-click the line and highlight only the block ID.

```
user-select: none
```

Reply:
That's super handy. Thanks